addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -29,4 +28,8 @@ export class FullscreenIndication extends Component {
             clearTimeout(this.autofade);
         }
     }
+
+    get fullScreenIndicationText() {
+        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
```

Comment:
```suggestion
        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), "<span>esc</span>"));
```
For previous version we have a fix coming https://github.com/odoo/odoo/pull/162757

Reply:
Oh, sorry for that 😕